core-services/openshift-priv/_whitelist.yaml

```
@@ -40,3 +42,4 @@ whitelist:
     - kubernetes-cli-runtime
     - kubernetes-sigs-structured-merge-diff
     - pj-rehearse # Temporary added. We will use this repo to test the publicize plugin.
+    - restic
```

```suggestion
    - openshift-velero-plugin
```
I completely forgot that we don't actually build a component from this repo; its binary is included as part of velero images.  Let's use [openshift-velero-plugin](https://github.com/openshift/openshift-velero-plugin) instead, please.